use-cases/tournament-scheduling/src/main/java/org/acme/tournamentschedule/rest/DemoDataGenerator.java

```diff
@@ -0,0 +1,80 @@
+package org.acme.tournamentschedule.rest;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+import org.acme.tournamentschedule.domain.Day;
+import org.acme.tournamentschedule.domain.Team;
+import org.acme.tournamentschedule.domain.TeamAssignment;
+import org.acme.tournamentschedule.domain.TournamentSchedule;
+import org.acme.tournamentschedule.domain.UnavailabilityPenalty;
+
+@ApplicationScoped
+public class DemoDataGenerator {
+
+    private final Random random = new Random(0);
+
+    public TournamentSchedule generateDemoData() {
+        TournamentSchedule schedule = new TournamentSchedule();
+        // Teams
+        List<Team> teams = generateTeams();
+        // Days
+        int countDays = 18;
+        List<Day> days = IntStream.range(0, countDays)
+                .mapToObj(Day::new)
+                .toList();
+        // Unavailability penalty
+        int countUnavailabilityPenalties = 12;
+        List<UnavailabilityPenalty> unavailabilityPenalties =
+                generateUnavailabilityPenalties(countUnavailabilityPenalties, teams, days);
+        // Assignments
+        int countAssignmentsPerDay = 4;
+        List<TeamAssignment> teamAssignments = generateTeamAssignments(countAssignmentsPerDay, days);
+        // Update schedule
+        schedule.setTeams(teams);
+        schedule.setDays(days);
+        schedule.setUnavailabilityPenalties(unavailabilityPenalties);
+        schedule.setTeamAssignments(teamAssignments);
+        return schedule;
+    }
+
+    private List<Team> generateTeams() {
+        return List.of(
+                new Team(0, "Micha"),
+                new Team(1, "Angelika"),
+                new Team(2, "Katrin"),
+                new Team(3, "Susi"),
+                new Team(4, "Irene"),
+                new Team(5, "Kristina"),
+                new Team(6, "Tobias"));
```

Comment:
Maybe we want to use our own names? :-) Maarten, Geoffrey, Lukas, Chris, Fred, Radek, Maciej, ...